internal/pkg/deploy/cloudformation/stack/transformers.go

```diff
@@ -67,7 +67,8 @@ var (
 )
 
 // convertSidecar converts the manifest sidecar configuration into a format parsable by the templates pkg.
-func convertSidecar(s map[string]*manifest.SidecarConfig) ([]*template.SidecarOpts, error) {
+func convertSidecar(s map[string]*manifest.SidecarConfig, portMappings []*template.PortMapping) ([]*template.SidecarOpts, error) {
```

Comment:
I know this isn't related to your change, but can we rename the `map` variable 😅 it doesn't tell us much. Perhaps just:
```suggestion
func convertSidecar(sidecars map[string]*manifest.SidecarConfig, portMappings []*template.PortMapping) ([]*template.SidecarOpts, error) {
```